TeacherWorkout.Api/Startup.cs

```diff
@@ -48,12 +56,27 @@ public void ConfigureServices(IServiceCollection services)
                 })
                 .AddErrorInfoProvider(opt => opt.ExposeExceptionStackTrace = true)
                 .AddSystemTextJson()
-                .AddGraphTypes();
+                .AddGraphTypes()
+                .AddUserContextBuilder(httpContext => new GraphQlUserContext(httpContext.User));
+            
+            services.AddControllers();
+
+            var applicationAssemblies = GetAssemblies();
+            services.AddSwaggerFor(applicationAssemblies, Configuration);
 
             services.AddDbContext<TeacherWorkoutContext>(options =>
                 options.UseNpgsql(Configuration.GetConnectionString("TeacherWorkoutContext")));
+
+            services.AddIdentityApiServices(Configuration);
```

Comment:
The initial discussion with @catileptic was that we should use IdentityServer (mentioned in the issue). From what I understand from the code, we are not using it. I am curious why the change.